src/utils/table.rb

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+# 表を表すクラス
+class Table
+  # @param [String] name 表の名前
+  # @param [String] type 表の振り方の種類 '1D6'など
+  # @param [Array] table 表本体
+  def initialize(name, type, table)
+    @name = name
+    @table = table.freeze
+
+    if (m = /(\d+)D(\d+)/i.match(type))
+      @type = :d
+      @times = m[1].to_i
+      @sides = m[2].to_i
+    else
+      raise "Unexpected table type: #{type}"
```

Comment:
例外の型としてはArgumentErrorが合いそうです。

```suggestion
      raise ArgumentError, "Unexpected table type: #{type}"
```